.github/workflows/ci_cd.yml

```diff
@@ -519,6 +520,56 @@ jobs:
       - name: Pass
         run: echo 'Django checks are skipped because API is unchanged.'
 
+  py-package-checks:
+    name: Run checks for Python packages/*
+    if: |
+      needs.get-changes.outputs.py_packages == 'true'
+    runs-on: ubuntu-latest
+    needs:
+      - get-changes
+      - lint
+    strategy:
+      fail-fast: false
+      matrix:
+        packages:
+          - packages/python/openverse-attribution
+        script:
+          - build
+          - test
```

Comment:
I wonder if these should be combined into a single `ci` script on all packages (JS ones included)?